### ISSUE TITLE
[Legitimate Site Blocked] metamese.com

### ISSUE THREAD

**Issue by stantechie** [MetaMask/eth-phishing-detect#6531]
https://metamese.com/
Another one of my domains for sale blocked by your extension.  Not cool.
This isn't even close to metamask.  There is nothing dangerous about selling a domain name.

**Comment by devenblake** [comment]
Duplicate of #6411.
